src/families/bitcoin/types.ts

```diff
@@ -35,17 +42,116 @@ export type BitcoinOutputRaw = [
   number,
   number | null | undefined,
   string | null | undefined,
+  number,
```

Comment:
ok to me this is a problem. we are changing the type `BitcoinOutputRaw ` which is stored in app.json.  it also obviously impact libcore impl too.

Reply:
Moved the new field to the end.